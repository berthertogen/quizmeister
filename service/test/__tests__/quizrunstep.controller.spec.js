test("should get question1 quizrunstep on Get (next)", async () => {
  await buildPostRequest("question", "", questionFactory.create());
  await buildPostRequest("question", "", questionFactory.create());
  await buildPostRequest("question", "", questionFactory.create());
  await buildPostRequest("question", "", questionFactory.create());
  await buildPostRequest("round", "", roundFactory.create([1,2]));
  await buildPostRequest("round", "", roundFactory.create([3,4]));
  await buildPostRequest("quiz", "", quizFactory.create([1,2]));
  await buildPutRequest("quiz", "1/status/5", {});
  await buildPostRequest("subscription", "", subscriptionFactory.create(1));

  const response = await buildGetRequest("quizrunstep", "1/next");
  assertContentTypeJsonAndStatus200(response);
  expect(response.body).toMatchSnapshot(quizrunstepFactory.ignorePropertiesFirstQuestion());
}); 

test("should get question2 quizrunstep on Get (next)", async () => {
  const response = await buildGetRequest("quizrunstep", "1/next");
  assertContentTypeJsonAndStatus200(response);
  expect(response.body).toMatchSnapshot(quizrunstepFactory.ignorePropertiesQuestion());
}); 

test("should get question4 quizrunstep on Get (current)", async () => {
  const response = await buildGetRequest("quizrunstep", "1/current");
  assertContentTypeJsonAndStatus200(response);
  expect(response.body).toMatchSnapshot(quizrunstepFactory.ignorePropertiesQuestion());
}); 

test("should get answers round 1 quizrunstep on Get (next)", async () => {
  const response = await buildGetRequest("quizrunstep", "1/next");
  assertContentTypeJsonAndStatus200(response);
  expect(response.body).toMatchSnapshot(quizrunstepFactory.ignorePropertiesAnswers());
}); 

test("should get scores round 1 quizrunstep on Get (next)", async () => {
  const response = await buildGetRequest("quizrunstep", "1/next");
  assertContentTypeJsonAndStatus200(response);
  expect(response.body).toMatchSnapshot(quizrunstepFactory.ignorePropertiesScores());
}); 

test("should get question3 quizrunstep on Get (next)", async () => {
  const response = await buildGetRequest("quizrunstep", "1/next");
  assertContentTypeJsonAndStatus200(response);
  expect(response.body).toMatchSnapshot(quizrunstepFactory.ignorePropertiesAfterScores());
}); 

test("should get question4 quizrunstep on Get (next)", async () => {
  const response = await buildGetRequest("quizrunstep", "1/next");
  assertContentTypeJsonAndStatus200(response);
  expect(response.body).toMatchSnapshot(quizrunstepFactory.ignorePropertiesQuestion());
}); 

test("should get answers round 2 quizrunstep on Get (next)", async () => {
  const response = await buildGetRequest("quizrunstep", "1/next");
  assertContentTypeJsonAndStatus200(response);
  expect(response.body).toMatchSnapshot(quizrunstepFactory.ignorePropertiesAnswers());
}); 

test("should get scores round 2 quizrunstep on Get (next)", async () => {
  const response = await buildGetRequest("quizrunstep", "1/next");
  assertContentTypeJsonAndStatus200(response);
  expect(response.body).toMatchSnapshot(quizrunstepFactory.ignorePropertiesScores());
}); 

test("should get null quizrunstep on Get (next)", async () => {
  const response = await buildGetRequest("quizrunstep", "1/next");
  assertContentTypeJsonAndStatus200(response);
  expect(response.body).toMatchSnapshot(quizrunstepFactory.ignorePropertiesFinish());
}); 


