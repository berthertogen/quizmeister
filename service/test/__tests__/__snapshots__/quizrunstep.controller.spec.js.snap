// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should get answers round 1 quizrunstep on Get (next) 1`] = `
Object {
  "message": null,
  "previousStep": Object {
    "answeredBy": Array [],
    "answers": null,
    "question": Object {
      "answers": Array [
        Object {
          "answerId": 3,
          "correct": true,
          "searchField": "trueanswer 1",
          "text": "answer 1",
        },
        Object {
          "answerId": 4,
          "correct": false,
          "searchField": "falseanswer 2",
          "text": "answer 2",
        },
      ],
      "modifiedOn": Any<String>,
      "questionId": 2,
      "remark": "remark",
      "rounds": Array [
        Object {
          "modifiedOn": Any<String>,
          "questions": null,
          "remark": "remark",
          "roundId": 1,
          "searchField": Any<String>,
          "shortId": Any<String>,
          "theme": "theme",
          "title": "title",
        },
      ],
      "scoring": Object {
        "scoringId": 2,
        "timeLimitSeconds": 0,
        "timeScoringInterval": 0,
        "type": 1,
        "weightCorrectAnswer": 1,
        "weightNoAnswer": 0,
      },
      "searchField": Any<String>,
      "shortId": Any<String>,
      "title": "title",
      "type": 2,
    },
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 2,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 1,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": null,
    "status": 1,
  },
  "status": 1,
  "step": Object {
    "answeredBy": null,
    "answers": Array [
      "answer 1",
      "answer 1",
    ],
    "question": null,
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 3,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 1,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": null,
    "status": 2,
  },
}
`;

exports[`should get answers round 2 quizrunstep on Get (next) 1`] = `
Object {
  "message": null,
  "previousStep": Object {
    "answeredBy": Array [],
    "answers": null,
    "question": Object {
      "answers": Array [
        Object {
          "answerId": 7,
          "correct": true,
          "searchField": "trueanswer 1",
          "text": "answer 1",
        },
        Object {
          "answerId": 8,
          "correct": false,
          "searchField": "falseanswer 2",
          "text": "answer 2",
        },
      ],
      "modifiedOn": Any<String>,
      "questionId": 4,
      "remark": "remark",
      "rounds": Array [
        Object {
          "modifiedOn": Any<String>,
          "questions": null,
          "remark": "remark",
          "roundId": 2,
          "searchField": Any<String>,
          "shortId": Any<String>,
          "theme": "theme",
          "title": "title",
        },
      ],
      "scoring": Object {
        "scoringId": 4,
        "timeLimitSeconds": 0,
        "timeScoringInterval": 0,
        "type": 1,
        "weightCorrectAnswer": 1,
        "weightNoAnswer": 0,
      },
      "searchField": Any<String>,
      "shortId": Any<String>,
      "title": "title",
      "type": 2,
    },
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 6,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 2,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": null,
    "status": 1,
  },
  "status": 1,
  "step": Object {
    "answeredBy": null,
    "answers": Array [
      "answer 1",
      "answer 1",
    ],
    "question": null,
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 7,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 2,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": null,
    "status": 2,
  },
}
`;

exports[`should get null quizrunstep on Get (next) 1`] = `
Object {
  "message": null,
  "previousStep": Object {
    "answeredBy": Array [],
    "answers": Array [
      "answer 1",
      "answer 1",
    ],
    "question": null,
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 8,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 2,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": Array [],
    "status": 3,
  },
  "status": 1,
  "step": Object {
    "answeredBy": null,
    "answers": null,
    "question": null,
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 9,
    "round": null,
    "scores": null,
    "status": 4,
  },
}
`;

exports[`should get question1 quizrunstep on Get (next) 1`] = `
Object {
  "message": null,
  "previousStep": null,
  "status": 1,
  "step": Object {
    "answeredBy": null,
    "answers": null,
    "question": Object {
      "answers": null,
      "modifiedOn": Any<String>,
      "questionId": 1,
      "remark": "remark",
      "rounds": Array [
        Object {
          "modifiedOn": Any<String>,
          "questions": null,
          "remark": "remark",
          "roundId": 1,
          "searchField": Any<String>,
          "shortId": Any<String>,
          "theme": "theme",
          "title": "title",
        },
      ],
      "scoring": Object {
        "scoringId": 1,
        "timeLimitSeconds": 0,
        "timeScoringInterval": 0,
        "type": 1,
        "weightCorrectAnswer": 1,
        "weightNoAnswer": 0,
      },
      "searchField": Any<String>,
      "shortId": Any<String>,
      "title": "title",
      "type": 2,
    },
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 1,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 1,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": null,
    "status": 1,
  },
}
`;

exports[`should get question2 quizrunstep on Get (next) 1`] = `
Object {
  "message": null,
  "previousStep": Object {
    "answeredBy": Array [],
    "answers": null,
    "question": Object {
      "answers": Array [
        Object {
          "answerId": 1,
          "correct": true,
          "searchField": "trueanswer 1",
          "text": "answer 1",
        },
        Object {
          "answerId": 2,
          "correct": false,
          "searchField": "falseanswer 2",
          "text": "answer 2",
        },
      ],
      "modifiedOn": Any<String>,
      "questionId": 1,
      "remark": "remark",
      "rounds": Array [
        Object {
          "modifiedOn": Any<String>,
          "questions": null,
          "remark": "remark",
          "roundId": 1,
          "searchField": Any<String>,
          "shortId": Any<String>,
          "theme": "theme",
          "title": "title",
        },
      ],
      "scoring": Object {
        "scoringId": 1,
        "timeLimitSeconds": 0,
        "timeScoringInterval": 0,
        "type": 1,
        "weightCorrectAnswer": 1,
        "weightNoAnswer": 0,
      },
      "searchField": Any<String>,
      "shortId": Any<String>,
      "title": "title",
      "type": 2,
    },
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 1,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 1,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": null,
    "status": 1,
  },
  "status": 1,
  "step": Object {
    "answeredBy": null,
    "answers": null,
    "question": Object {
      "answers": null,
      "modifiedOn": Any<String>,
      "questionId": 2,
      "remark": "remark",
      "rounds": Array [
        Object {
          "modifiedOn": Any<String>,
          "questions": null,
          "remark": "remark",
          "roundId": 1,
          "searchField": Any<String>,
          "shortId": Any<String>,
          "theme": "theme",
          "title": "title",
        },
      ],
      "scoring": Object {
        "scoringId": 2,
        "timeLimitSeconds": 0,
        "timeScoringInterval": 0,
        "type": 1,
        "weightCorrectAnswer": 1,
        "weightNoAnswer": 0,
      },
      "searchField": Any<String>,
      "shortId": Any<String>,
      "title": "title",
      "type": 2,
    },
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 2,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 1,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": null,
    "status": 1,
  },
}
`;

exports[`should get question3 quizrunstep on Get (next) 1`] = `
Object {
  "message": null,
  "previousStep": Object {
    "answeredBy": Array [],
    "answers": Array [
      "answer 1",
      "answer 1",
    ],
    "question": null,
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 4,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 1,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": Array [],
    "status": 3,
  },
  "status": 1,
  "step": Object {
    "answeredBy": null,
    "answers": null,
    "question": Object {
      "answers": null,
      "modifiedOn": Any<String>,
      "questionId": 3,
      "remark": "remark",
      "rounds": Array [
        Object {
          "modifiedOn": Any<String>,
          "questions": null,
          "remark": "remark",
          "roundId": 2,
          "searchField": Any<String>,
          "shortId": Any<String>,
          "theme": "theme",
          "title": "title",
        },
      ],
      "scoring": Object {
        "scoringId": 3,
        "timeLimitSeconds": 0,
        "timeScoringInterval": 0,
        "type": 1,
        "weightCorrectAnswer": 1,
        "weightNoAnswer": 0,
      },
      "searchField": Any<String>,
      "shortId": Any<String>,
      "title": "title",
      "type": 2,
    },
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 5,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 2,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": null,
    "status": 1,
  },
}
`;

exports[`should get question4 quizrunstep on Get (current) 1`] = `
Object {
  "message": null,
  "previousStep": Object {
    "answeredBy": Array [],
    "answers": null,
    "question": Object {
      "answers": Array [
        Object {
          "answerId": 1,
          "correct": true,
          "searchField": "trueanswer 1",
          "text": "answer 1",
        },
        Object {
          "answerId": 2,
          "correct": false,
          "searchField": "falseanswer 2",
          "text": "answer 2",
        },
      ],
      "modifiedOn": Any<String>,
      "questionId": 1,
      "remark": "remark",
      "rounds": Array [
        Object {
          "modifiedOn": Any<String>,
          "questions": null,
          "remark": "remark",
          "roundId": 1,
          "searchField": Any<String>,
          "shortId": Any<String>,
          "theme": "theme",
          "title": "title",
        },
      ],
      "scoring": Object {
        "scoringId": 1,
        "timeLimitSeconds": 0,
        "timeScoringInterval": 0,
        "type": 1,
        "weightCorrectAnswer": 1,
        "weightNoAnswer": 0,
      },
      "searchField": Any<String>,
      "shortId": Any<String>,
      "title": "title",
      "type": 2,
    },
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 1,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 1,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": null,
    "status": 1,
  },
  "status": 1,
  "step": Object {
    "answeredBy": Array [],
    "answers": null,
    "question": Object {
      "answers": Array [
        Object {
          "answerId": 3,
          "correct": true,
          "searchField": "trueanswer 1",
          "text": "answer 1",
        },
        Object {
          "answerId": 4,
          "correct": false,
          "searchField": "falseanswer 2",
          "text": "answer 2",
        },
      ],
      "modifiedOn": Any<String>,
      "questionId": 2,
      "remark": "remark",
      "rounds": Array [
        Object {
          "modifiedOn": Any<String>,
          "questions": null,
          "remark": "remark",
          "roundId": 1,
          "searchField": Any<String>,
          "shortId": Any<String>,
          "theme": "theme",
          "title": "title",
        },
      ],
      "scoring": Object {
        "scoringId": 2,
        "timeLimitSeconds": 0,
        "timeScoringInterval": 0,
        "type": 1,
        "weightCorrectAnswer": 1,
        "weightNoAnswer": 0,
      },
      "searchField": Any<String>,
      "shortId": Any<String>,
      "title": "title",
      "type": 2,
    },
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 2,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 1,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": null,
    "status": 1,
  },
}
`;

exports[`should get question4 quizrunstep on Get (next) 1`] = `
Object {
  "message": null,
  "previousStep": Object {
    "answeredBy": Array [],
    "answers": null,
    "question": Object {
      "answers": Array [
        Object {
          "answerId": 5,
          "correct": true,
          "searchField": "trueanswer 1",
          "text": "answer 1",
        },
        Object {
          "answerId": 6,
          "correct": false,
          "searchField": "falseanswer 2",
          "text": "answer 2",
        },
      ],
      "modifiedOn": Any<String>,
      "questionId": 3,
      "remark": "remark",
      "rounds": Array [
        Object {
          "modifiedOn": Any<String>,
          "questions": null,
          "remark": "remark",
          "roundId": 2,
          "searchField": Any<String>,
          "shortId": Any<String>,
          "theme": "theme",
          "title": "title",
        },
      ],
      "scoring": Object {
        "scoringId": 3,
        "timeLimitSeconds": 0,
        "timeScoringInterval": 0,
        "type": 1,
        "weightCorrectAnswer": 1,
        "weightNoAnswer": 0,
      },
      "searchField": Any<String>,
      "shortId": Any<String>,
      "title": "title",
      "type": 2,
    },
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 5,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 2,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": null,
    "status": 1,
  },
  "status": 1,
  "step": Object {
    "answeredBy": null,
    "answers": null,
    "question": Object {
      "answers": null,
      "modifiedOn": Any<String>,
      "questionId": 4,
      "remark": "remark",
      "rounds": Array [
        Object {
          "modifiedOn": Any<String>,
          "questions": null,
          "remark": "remark",
          "roundId": 2,
          "searchField": Any<String>,
          "shortId": Any<String>,
          "theme": "theme",
          "title": "title",
        },
      ],
      "scoring": Object {
        "scoringId": 4,
        "timeLimitSeconds": 0,
        "timeScoringInterval": 0,
        "type": 1,
        "weightCorrectAnswer": 1,
        "weightNoAnswer": 0,
      },
      "searchField": Any<String>,
      "shortId": Any<String>,
      "title": "title",
      "type": 2,
    },
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 6,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 2,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": null,
    "status": 1,
  },
}
`;

exports[`should get scores round 1 quizrunstep on Get (next) 1`] = `
Object {
  "message": null,
  "previousStep": Object {
    "answeredBy": Array [],
    "answers": Array [
      "answer 1",
      "answer 1",
    ],
    "question": null,
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 3,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 1,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": null,
    "status": 2,
  },
  "status": 1,
  "step": Object {
    "answeredBy": Array [],
    "answers": Array [
      "answer 1",
      "answer 1",
    ],
    "question": null,
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 4,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 1,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": Array [],
    "status": 3,
  },
}
`;

exports[`should get scores round 2 quizrunstep on Get (next) 1`] = `
Object {
  "message": null,
  "previousStep": Object {
    "answeredBy": Array [],
    "answers": Array [
      "answer 1",
      "answer 1",
    ],
    "question": null,
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 7,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 2,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": null,
    "status": 2,
  },
  "status": 1,
  "step": Object {
    "answeredBy": Array [],
    "answers": Array [
      "answer 1",
      "answer 1",
    ],
    "question": null,
    "quiz": Object {
      "date": "2100-01-31T23:00:00",
      "location": "location",
      "maxSubscriptions": 20,
      "modifiedOn": Any<String>,
      "quizId": 1,
      "remark": "remark",
      "rounds": null,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "status": 5,
      "subscriptions": null,
      "title": "title",
    },
    "quizRunStepId": 8,
    "round": Object {
      "modifiedOn": Any<String>,
      "questions": null,
      "remark": "remark",
      "roundId": 2,
      "searchField": Any<String>,
      "shortId": Any<String>,
      "theme": "theme",
      "title": "title",
    },
    "scores": Array [],
    "status": 3,
  },
}
`;
